lib/internal/test_runner/coverage.js

```diff
@@ -477,7 +477,7 @@ function sortCoverageFiles(a, b) {
 
 function setupCoverage(options) {
   let originalCoverageDirectory = process.env.NODE_V8_COVERAGE;
-  const cwd = process.cwd();
+  const cwd = options.cwd ?? process.cwd();
```

Comment:
We should ensure that `options.cwd` always exists and remove the `?? process.cwd()` fallback.